src/styles/main.scss

```
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import "./utilits/_normalize";
+@import "./blocks/header";
+@import "./blocks/_nav";
+@import "./blocks/_catalog";
+@import "./blocks/_stars";
+@import "./blocks/_card";
```

it should work this way
```suggestion
@import "./utilits/normalize";
@import "./blocks/header";
@import "./blocks/nav";
@import "./blocks/catalog";
@import "./blocks/stars";
@import "./blocks/card";
```